lib/Runtime/Library/WebAssemblyMemory.h

```diff
@@ -9,6 +9,9 @@ namespace Js
 {
     class WebAssemblyMemory : public DynamicObject
     {
+    protected:
+        DEFINE_VTABLE_CTOR( WebAssemblyMemory, DynamicObject );
+        DEFINE_MARSHAL_OBJECT_TO_SCRIPT_CONTEXT( WebAssemblyMemory );
```

Comment:
ok. I think this is fine to add, but we may still have more to do for marshalling to really work. We can talk to Curtis in January for that though.